src/models/piranha_output.rs

```diff
@@ -45,25 +53,10 @@ impl PiranhaOutputSummary {
   pub(crate) fn new(source_code_unit: &SourceCodeUnit) -> PiranhaOutputSummary {
     return PiranhaOutputSummary {
       path: String::from(source_code_unit.path().as_os_str().to_str().unwrap()),
+      original_content: source_code_unit.original_content().to_string(),
       content: source_code_unit.code().to_string(),
       matches: source_code_unit.matches().iter().cloned().collect_vec(),
       rewrites: source_code_unit.rewrites().iter().cloned().collect_vec(),
     };
   }
-
-  pub(crate) fn matches(&self) -> &[(String, Match)] {
```

Comment:
What happened to these functionalities?

Reply:
We dont need them. These have been replaced by the annotation `#[get = "pub(crate)"]` which is added to corresponding property.